src/renderer/components/ft-element-list/ft-element-list.vue

```diff
@@ -4,7 +4,7 @@
   >
     <ft-list-lazy-wrapper
       v-for="(result, index) in data"
-      :key="index"
+      :key="`${result.type}-${result.videoId || result.playlistId || result.id}-${index}`"
```

Comment:
Community posts have `postId` and channel items returned by the Invidious API use `authorId` as their main id, however as videos, playlists and community posts all also have an `authorId` we need to make sure to have it right at the end of the list.

```suggestion
      :key="`${result.type}-${result.videoId || result.playlistId || result.postId || result.id || result.authorId}-${index}`"
```